action-helm-tools/main.sh

```diff
@@ -31,9 +31,16 @@ set_commit_status() {
   STATUS=$1
 
   if [[ "$GITHUB_EVENT_NAME" == "workflow_dispatch" ]] && [[ -n "$GITHUB_SHA" ]] && [[ -n "$GITHUB_TOKEN" ]] && [[ -n "$GITHUB_REPOSITORY" ]]; then
+
+    if [ -n "$COMMITSHA" ]; then
```

Comment:
If there is a "commitsha" workflow input we will use that to set commit status to